guide/payments/send.md

```diff
@@ -0,0 +1,161 @@
+
+**QR Code** -- Once the camera detects a valid address in the QR Code, it can be automatically inputted. The user will need to grant access your application to be able to scan QR Codes.
+
+**Copy Paste** -- When the sender receives the address in text format they would need to paste in the address. Your application can detect a valid address in the clipboard and prompt the user to press a button to paste.
+
+![](https://i.imgur.com/M9ajQz9.png)
+
+**Do's**
+
+- Indicate clearly if the address is valid or not
+- Show the whole address if possible to help the user visually verify it is correct
+- If space is a constraint, truncate the address in the middle so that both the beginning and end are visible
+- Most platforms allow applications to read the contents of the device's clipboard to enable convenient copy/paste. While some users find this helpful, some may perceive it as a privacy infringement. You may want to consider giving them an option to disable automatic reading of the clipboard.
+
+**Don'ts**
+
+- Don't allow a transaction to be sent if the address is invalid
+
+## Inputing an amount
+
+Depending on their familiarity with bitcoin, your users may have a preference to which denomination the amount should be specified in. Common options include:
+
+- **bitcoin** — most common *(i.e. 0.00019376 btc)*
+- **local currency** — more familiar *(i.e. $10)*
+- **sats / satoshis** — more knowledgeable *(i.e. 19376 sats)*
+
+Allowing the amount to be inputed in different denominations should be readily available. Read more about why and changing units contextually in [Units, symbols and amount display](https://deploy-preview-63--sad-borg-390916.netlify.app/guide/payments/units-and-symbols/).
+
+Since an amount specified in bitcoin or sats often involve many digits, and don't correspond easily to a local currency amount ($10 as 0.00019376 bitcoin for example). Applications can allow the sender select fractions of the total available balance. Most common is *max* or *all*, which also makes it easy for the user to sweep the entire balance to another wallet.
+
+![](https://i.imgur.com/vSYVVLu.png)
+
+**Do's**
+
+- Allow different denominations to be used for the amount
+- Show the total balance available to send
+- Allow selecting max amount (subtracting the required fee)
+- Payment links and QR codes can contain an amount — when they do, the amount should be populated automatically.
+
+**Don'ts**
+
+- Don't allow entering more than the available balance
+
+
+
+## Transaction fee
+
+The application can automatically estimate a fee and set it for the sender. This would normally prioritize the transaction to be included in a block as soon as possible. Since the fee rate may vary if the network is busy, you can give senders more fine grained fee controls so they can choose to optimize for faster confirmation, or lower fees.
```

Comment:
```suggestion
The application can automatically estimate a fee and set it for the sender. This would normally prioritize the transaction to be included in a block as soon as possible. Since the fee rate may vary if the network is busy, you can give senders more fine-grained fee controls so they can choose to optimize for faster confirmation or lower fees.
```